vendir.lock

```diff
@@ -0,0 +1,15 @@
+apiVersion: vendir.k14s.io/v1alpha1
```

Comment:
this file can be deleted probably. there is also the file `vendir.lock.yml` with changes